src/test/groovy/graphql/parser/ParserTest.groovy

```diff
@@ -516,17 +516,18 @@ class ParserTest extends Specification {
     }
 
 
-    def "parses null value"() {
+    def "parses null values"() {
         given:
-        def input = "{ foo(bar: null) }"
+        def input = "{ foo(bar: null, bell : null) }"
 
         when:
         def document = new Parser().parseDocument(input)
         def operation = document.definitions[0] as OperationDefinition
         def selection = operation.selectionSet.selections[0] as Field
 
         then:
-        selection.arguments[0].value == NullValue.Null
+        assert selection.arguments[0].value instanceof NullValue
```

Comment:
please make sure that source location is also set

Reply:
The constant has been deleted - see changes to NullValue file. 

What do you mean by source location is set?